### DISCUSSION TITLE
Connecting to GNS3 Server

### DISCUSSION THREAD

**Discussion by northernscumiam** [GNS3/gns3-gui#3562] [General]
Hi there
I'm rather new to the GNS world and wondering how I would connect my Windows laptop to our GNS3 server?
Any assistance would be amazing. Thanks

**Comment by geryescalier** [comment]
Hello @northernscumiam  this video https://www.youtube.com/watch?v=Bo7VwB3KZZo can help you connect your gns3 client (your laptop) with a gns3 server
Regards!
